mw-webapp/src/model/businessModel/MonthReport.ts

```diff
@@ -34,9 +35,9 @@ export class MonthReport {
   public problemsForCurrentPeriod: CurrentProblem[];
 
   /**
-   * Anything that student wants to say about work
+   * Array of student comments and their status
```

Comment:
you have typescript to handle types, and fields name don't use docs for that

Reply:
Done